### DISCUSSION TITLE
Helm: memberlist bind address

### DISCUSSION THREAD

**Discussion by thel0** [grafana/mimir#3470] [Help and support]
hello
i try to install mimir with helm and it's not so easy :) 

All pod does not find their IP but with this forum i find the solution with :
    extraArgs:
      memberlist.bind-addr: ${MY_POD_IP}

Is the slack forum always UP ? i try to join slack.grafana.com and the URL is in error since few days

thanks for your help
thel

**Comment by pracucci** [comment]
> All pod does not find their IP but with this forum i find the solution with :
> extraArgs:
> memberlist.bind-addr: ${MY_POD_IP}

I'm not sure if it's a question or a statement. Do you have any specific question?

> Is the slack forum always UP ? i try to join slack.grafana.com and the URL is in error since few days

Thanks for the report. I'm escalating it at Grafana Labs.
